org.gnu.Octave.json

```diff
@@ -0,0 +1,298 @@
+{
+	"app-id": "org.gnu.Octave",
+	"runtime": "org.kde.Sdk",
+	"runtime-version": "5.9",
+	"sdk": "org.kde.Sdk",
+	"command": "octave",
+	"rename-icon": "octave",
+	"rename-desktop-file": "www.octave.org-octave.desktop",
+	"rename-appdata-file": "www.octave.org-octave.appdata.xml",
+	"finish-args":
+	[
+		"--socket=wayland",
+		"--socket=x11",
+		"--socket=pulseaudio",
+		"--share=network",
+		"--device=dri",
+		"--filesystem=home"
+	],
+	"build-options":
+	{
+		"cflags": "-O2",
+		"cxxflags": "-O2"
+	},
+	"modules":
+	[
+		{
+			"name": "gcc-fortran",
+			"build-options":
+			{
+				"ldflags": "--static-libgcc"
+			},
+			"config-opts":
+			[
+				"--enable-languages=c,c++,fortran",
+				"--disable-multilib",
+				"--disable-bootstrap",
+				"--with-system-zlib",
+				"--enable-checking=release",
+				"--disable-stage1-checking",
+				"--disable-libsanitizer",
+				"--disable-libssp"
+			],
+			"sources":
+			[
+				{
+					"type": "archive",
+					"url": "http://mirrors.kernel.org/gnu/gcc/gcc-7.1.0/gcc-7.1.0.tar.gz",
+					"sha256": "3c187ef42e4cf657eec0ed641aac7f7e46261bcb1cea31b72dcf2dedddbbdea5"
+				},
+				{
+					"type": "shell",
+					"commands":
+					[
+						"sed -e '/m64=/s/lib64/lib/' -i.orig gcc/config/i386/t-linux64"
+					]
+				}
+			]
+		},
+		{
+			"name": "openblas",
+			"no-autogen": true,
+			"build-options":
+			{
+				"ldflags": "-L/app/lib64"
+			},
+			"make-args":
+			[
+				"FC=gfortran",
+				"TARGET=PRESCOTT"
+			],
+			"make-install-args":
+			[
+				"PREFIX=/app"
+			],
+			"sources":
+			[
+				{
+					"type": "archive",
+					"url": "https://github.com/xianyi/OpenBLAS/archive/v0.2.19.tar.gz",
+					"sha256": "9c40b5e4970f27c5f6911cb0a28aa26b6c83f17418b69f8e5a116bb983ca8557"
+				}
+			]
+		},
+		{
+			"name": "qscintilla",
+			"buildsystem": "simple",
+			"sources":
+			[
+				{
+					"type": "archive",
+					"url": "https://sourceforge.net/projects/pyqt/files/QScintilla2/QScintilla-2.10/QScintilla_gpl-2.10.tar.gz",
+					"sha256": "16be30577bc178470936c458551f2512cc068aff6e7a7de6ed244e28c045f6ec"
+				},
+				{
+					"type": "patch",
+					"path": "qscintilla.patch"
+				},
+				{
+					"type": "script",
+					"commands":
+					[
+						"cd Qt4Qt5",
+						"qmake"
+					],
+					"dest-filename": "configure"
+				}
+			],
+			"build-commands":
+			[
+				"./configure --prefix=/app && \
+				cd Qt4Qt5 && \
+				make && \
+				make install && \
```

Comment:
Can simplify this into `make -C Qt4Qt5 install`. Also don't continue lines just have separate strings.